includes/migration-guide/retargeting/versionselector.md

```diff
@@ -1,29 +1,37 @@
 > [!div class="op_multi_selector" title1="From .NET Framework Version" title2="To .NET Framework Version"]
-> - [(4.0 | 4.5)](~/docs/framework/migration-guide/retargeting/4.0-4.5.md)
-> - [(4.0 | 4.5.1)](~/docs/framework/migration-guide/retargeting/4.0-4.5.1.md)
-> - [(4.0 | 4.5.2)](~/docs/framework/migration-guide/retargeting/4.0-4.5.2.md)
-> - [(4.0 | 4.6)](~/docs/framework/migration-guide/retargeting/4.0-4.6.md)
-> - [(4.0 | 4.6.1)](~/docs/framework/migration-guide/retargeting/4.0-4.6.1.md)
-> - [(4.0 | 4.6.2)](~/docs/framework/migration-guide/retargeting/4.0-4.6.2.md)
-> - [(4.0 | 4.7)](~/docs/framework/migration-guide/retargeting/4.0-4.7.md)
-> - [(4.5 | 4.5.1)](~/docs/framework/migration-guide/retargeting/4.5-4.5.1.md)
-> - [(4.5 | 4.5.2)](~/docs/framework/migration-guide/retargeting/4.5-4.5.2.md)
-> - [(4.5 | 4.6)](~/docs/framework/migration-guide/retargeting/4.5-4.6.md)
-> - [(4.5 | 4.6.1)](~/docs/framework/migration-guide/retargeting/4.5-4.6.1.md)
-> - [(4.5 | 4.6.2)](~/docs/framework/migration-guide/retargeting/4.5-4.6.2.md)
+> - [(4.0 | 4.5)](../../../../docs/framework/migration-guide/retargeting/4.0-4.5.md)
+> - [(4.0 | 4.5.1)](../../../../docs/framework/migration-guide/retargeting/4.0-4.5.1.md)
+> - [(4.0 | 4.5.2)](../../../../docs/framework/migration-guide/retargeting/4.0-4.5.2.md)
+> - [(4.0 | 4.6)](../../../../docs/framework/migration-guide/retargeting/4.0-4.6.md)
+> - [(4.0 | 4.6.1)](../../../../docs/framework/migration-guide/retargeting/4.0-4.6.1.md)
+> - [(4.0 | 4.6.2)](../../../../docs/framework/migration-guide/retargeting/4.0-4.6.2.md)
+> - [(4.0 | 4.7)](../../../../docs/framework/migration-guide/retargeting/4.0-4.7.md)
+> - [(4.0 | 4.7.1)](../../../../docs/framework/migration-guide/retargeting/4.0-4.7.1.md)
+> - [(4.5 | 4.5.1)](../../../../docs/framework/migration-guide/retargeting/4.5-4.5.1.md)
+> - [(4.5 | 4.5.2)](../../../../docs/framework/migration-guide/retargeting/4.5-4.5.2.md)
+> - [(4.5 | 4.6)](../../../../docs/framework/migration-guide/retargeting/4.5-4.6.md)
+> - [(4.5 | 4.6.1)](../../../../docs/framework/migration-guide/retargeting/4.5-4.6.1.md)
+> - [(4.5 | 4.6.2)](../../../../docs/framework/migration-guide/retargeting/4.5-4.6.2.md)
```

Comment:
Connie, these links are all broken. The original versions (~/docs/framework/migration-guide... etc.) all work,